pyproject.toml

```diff
@@ -368,6 +349,10 @@ exclude = [
 # Keep this one enabled
 "pandas/_typing.py" = ["TCH"]
 
+[tool.ruff.format]
+docstring-code-format = true
```

Comment:
Personally, I like to replace as much with ruff as possible :) but it might be faster to have one PR to replace black and a second one for the doc formatting (I believe black isn't used for that?).

Reply:
Good point. Opened up https://github.com/pandas-dev/pandas/pull/56704 to just replace black